src/status_im/ui/screens/wallet/navigation.cljs

```diff
@@ -7,15 +7,21 @@
 (defmethod navigation/preload-data! :wallet
   [db _]
   ;;TODO(goranjovic) - get rid of this preload hook completely
-  (re-frame/dispatch [:wallet.ui/pull-to-refresh])
-  (re-frame/dispatch [:update-wallet])
+  (js/setTimeout
```

Comment:
is that a part of subs optimizations?

Reply:
no, just temporary workaround for wallet screen

Reply:
could you remove it and put it in a separate with the reasoning behind it?